src/UpdateHTML.ts

```diff
@@ -615,6 +615,13 @@ export const buttoncolorchange = () => {
         DOMCacheGetOrSet('ascendChallengeBtn').style.backgroundColor = '#171717' :
         DOMCacheGetOrSet('ascendChallengeBtn').style.backgroundColor = 'purple';
 
+    (player.autoAscend && player.cubeUpgrades[10] > 0.5) ?
+        DOMCacheGetOrSet('ascendbtn').style.backgroundColor = 'green' :
+        DOMCacheGetOrSet('ascendbtn').style.backgroundColor = '#171717';
+
+    (player.singularityCount > 0.5)
+    DOMCacheGetOrSet('singularitybtn').style.backgroundColor = '#171717';
```

Comment:
is this intended?

Reply:
Its makes possible to have no BG while and light mode not lookin weird with Singularity

Reply:
![image](https://user-images.githubusercontent.com/107661324/174514488-87553486-eb11-4677-a03f-ec36d8a739b6.png)
Here what will happen if to remove these lines